bogota-july-2018/javascript-calculator/Felipe Jimenez/index.html

```diff
@@ -0,0 +1,32 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Calculator</title>
+	  <script type="text/javascript" src="js/script.js"></script>
+    <link href="css/style.css" rel="stylesheet">
+  </head>
+  <body>
+    <header>
+      <h1>JS Calculator</h1>
+    </header>
+    <main>
+      <p id="board">0</p>
+      <form>
+        <label for="numberLeft">First Number</label>
+        <input id="numberLeft" type="text" class="calculatorElement"><br>
+
+        <label for="operator">Operator</label>
```

Comment:
Well this doesnt looks like a clasic calculator. Please add all the numbers and a classic display.